rs/cli/src/auth.rs

```diff
@@ -90,7 +250,7 @@ impl Neuron {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
I always found it funny that Rust doesn't by default make types comparable, but when I learned that this provides the Rust compiler some opportunities for serious optimization quite literally impossible in other languages, I understood why.